src/components/pages/HomePage.js

```diff
@@ -0,0 +1,311 @@
+                                    </Form.Group>
+                                </Form>
+                            </Modal.Description>
+                        </Modal>
+                    </Menu.Menu>
+                </Menu>
+            </div>
+            <Grid style={Styles.header} centered padded relaxed>
+            <Grid.Row>
+            </Grid.Row>
+             <Grid.Row>
+             <Link to='/requestPipeline'>
+             <Button size='large'>Make a Request</Button>
+             </Link>
+            </Grid.Row>
+
+             <Grid.Row>
+            </Grid.Row>
+
+            </Grid>
+            </div>
+    );
+  }
+}
+;
+
+const mapDispatchToProps = dispatch => {
+  return {
+    setEmail: event => {
+      const value = event.target.value;
+      const action = {
+          type: Actions.login.setEmail,
+          value
+        };
+
+      dispatch(action);
+    },
+    setPassword: event => {
+      const value = event.target.value;
+      const action = {
+          type: Actions.login.setPassword,
+          value
+        };
+
+      dispatch(action);
+    },
+    setFirstName: event => {
+      const value = event.target.value;
+      const action = {
+          type: Actions.login.setFirstName,
+          value
+        };
+
+      dispatch(action);
+    },
+    setLastName: event => {
+      const value = event.target.value;
+      const action = {
+          type: Actions.login.setLastName,
+          value
+        };
+
+      dispatch(action);
+    },
+    setPasswordRepeat: event => {
+      const value = event.target.value;
+      const action = {
+          type: Actions.login.setPasswordRepeat,
+          value
+        };
+
+      dispatch(action);
+    },
+    setAddress: event => {
+      const value = event.target.value;
+      const action = {
+          type: Actions.login.setAddress,
+          value
+        };
+
+      dispatch(action);
+    },
+    setCity: event => {
+      const value = event.target.value;
+      const action = {
+          type: Actions.login.setCity,
+          value
+        };
+
+      dispatch(action);
+    },
+    setAddressState: (event, data) => {
+      const value = data.value;
+      const action = {
+          type: Actions.login.setAddressState,
+          value
+        };
+
+      dispatch(action);
+    },
+    setZipcode: event => {
+      const value = event.target.value;
+      const action = {
+          type: Actions.login.setZipcode,
+          value
+        };
+
+      dispatch(action);
+    },
+    openLoginModal: () => {
+      const action = {
+        type: Actions.login.toggleLoginModal
+      };
+
+      dispatch(action);
+    },
+    closeLoginModal: () => {
+      const action = {
+        type: Actions.login.clear
+      };
+
+      dispatch(action);
+    },
+    openSignupModal: () => {
+      const action = {
+        type: Actions.login.toggleSignupModal
+      };
+
+      dispatch(action);
+    },
+    closeSignupModal: () => {
```

Comment:
It seems a lot of these dispatched actions essentially do the same thing. Shouldn't we be able to create a single action to change the state regarding all strings?

Reply:
That will need to be for another story. This is more related to the button and that is with the login and signup modals